src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,72 @@
+package commands;
+
+import functional.DukeException;
+import functional.TaskList;
+import functional.Ui;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDo;
+
+import java.util.Arrays;
+
+public class AddCommand extends commands.Command {
```

Comment:
Any reason why you need to extends commands.Command rather than just extends Command?